inc/namespace.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace HM\Unpublish;
+
+const CRON_KEY = 'unpublish_post_cron';
+const FEATURE_NAME = 'unpublish';
+const POST_META_KEY = 'unpublish_timestamp';
+const VERSION = '2.0.0-alpha1';
+
+/**
+ * Bootstrapper
+ *
+ * @return void
+ */
+function bootstrap(): void {
+	add_action( CRON_KEY, __NAMESPACE__ . '\\unpublish_post' );
+	add_action( 'trashed_post', __NAMESPACE__ . '\\unschedule_unpublish' );
+	add_action( 'untrashed_post', __NAMESPACE__ . '\\reschedule_unpublish' );
+}
+
+/**
+ * Get unpublish timestamp
+ *
+ * @param int $post_id Post ID.
+ *
+ * @return int|null Unpublish timestamp if set. NULL otherwise.
+ */
+function get_unpublish_timestamp( int $post_id ) : ? int {
+	$value = get_post_meta( $post_id, POST_META_KEY, true );
+	$value = ! empty( $value ) ? absint( $value ) : null;
+
+	return $value;
+}
+
+/**
+ * Unschedule unpublishing post
+ *
+ * @param int $post_id Post ID.
+ *
+ * @return mixed On success an integer indicating number of events unscheduled
+ *               (0 indicates no events were registered with the hook and
+ *               arguments combination), false if unscheduling one or more events fail.
+ */
+function unschedule_unpublish( int $post_id ) {
+	return wp_clear_scheduled_hook( CRON_KEY, array( $post_id ) );
```

Comment:
There are few more instances of multiple array declarations, perhaps this can be automated using phpcs rule, in a different issue?

```
<rule ref="Generic.Arrays.DisallowLongArraySyntax.Found">
	<type>warning</type>
</rule>
```

Reply:
I see that this is done as a part of #35 already.